build-release.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+JAVA_HOME=/Applications/Android\ Studio.app/Contents/jre/Contents/Home ANDROID_HOME=~/Library/Android/sdk ./gradlew clean publishReleasePublicationToMavenRepository
```

Comment:
Should this file be part of the release?

Reply:
ya I realized I hadn't documented the build anywhere in this particular repo